org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/servicetemplates/topologytemplates/TopologyTemplateResource.java

```diff
@@ -205,9 +208,9 @@ public Response triggerGenerateBuildPlan(@Context UriInfo uriInfo) {
 		Client client = Client.create();
 
 		Builder wr = null;
-		if (RestUtils.isResourceAvailable("http://localhost:1339/planbuilder")) {
```

Comment:
The new structure of Winery uses `RestUtils` instead of `Utils`. Please follow that.